src/main/java/com/microsoft/azure/relay/HybridConnectionListener.java

```diff
@@ -277,50 +276,46 @@ public HybridConnectionListener(String connectionString, String path) throws URI
 	 * @return A CompletableFuture which completes when all connections are
 	 *         disconnected with the cloud service
 	 */
+	@SuppressWarnings("unchecked")
```

Comment:
Can this suppress be on the specific line which requires it?

[http://thefinestartist.com/effective-java/24](http://thefinestartist.com/effective-java/24)
"Always use the SuppressWarnings annotation on the smallest scope possible"

Reply:
I can't figure out a way to make it any smaller scoped.

---
In reply to: [257777052](https://github.com/Azure/azure-relay-java/pull/14#discussion_r257777052) [](ancestors = 257777052)